### DISCUSSION TITLE
Orphaned cake-contrib projects

### DISCUSSION THREAD

**Discussion by mcliment** [cake-build/cake#3738] [Q&A] [answered]
Hi all!

Some people are trying to move forward https://github.com/cake-contrib/Cake.Sonar but the maintainer (@tomstaijen) seems unavailable for a long time so PRs are not approved, nothing can be merged and the project can't move forward.

What is the procedure on these cases? Can someone else take ownership? I could do it myself and check the existing PRs as well as fix some of the issues and release a new version.

Thanks for your help :)

**Comment by tomstaijen** [comment]
I'll try to add you Marc. And perhaps we can find one more volunteer.

**Reply by gep13** [reply]
@tomstaijen if you are happy, I can get @mcliment added to the Cake-Contrib GitHub Organisation, and give him permissions to the Cake.Sonar repository?

**Comment by tomstaijen** [comment, marked as answer]
Go for it!

**Reply by gep13** [reply]
@mcliment I have sent you an invite to join the GitHub Organisation.

**Reply by tomstaijen** [reply]
Let's add @HofmeisterAn also, he's volunteering and it's good to not depend on a single person.

**Reply by mcliment** [reply]
Invite accepted, thanks!

**Reply by gep13** [reply]
@tomstaijen I have sent an invite through to @HofmeisterAn as well.  Thanks for the suggestion.

**Reply by HofmeisterAn** [reply]
@gep13 Thanks for the invite. May it be possible to get write access to the repository? Then @mcliment and I can merge pull requests like cake-contrib/Cake.Sonar#111 and close the related issues.

**Reply by gep13** [reply]
@HofmeisterAn thank you for your help with this project.  You should now have admin access to this repository. Let me know if you need anything else.

**Reply by HofmeisterAn** [reply]
@gep13 Thanks, I was now able to merge the pull request. Unfortunately, GitHub is not able to execute the webhook and queue the build successfully. I get `Push request and project repositories do not match.`. The webhook endpoint is: `https://ci.appveyor.com/api/github/webhook?id=sm1....7rac` 

It looks like the Cake Contributions organization manages the pipeline secrets. Do we need to update the webhook URL too? Any idea what goes wrong? Sorry to bother you with that.

**Reply by gep13** [reply]
@HofmeisterAn an AppVeyor project under the Cake-Contrib Organisation had never been created.  The existing webhooks must have been from when the project was hosted elsewhere.  I have added a new project, and the webhook now seems to be working as expected.

I will need to get the AppVeyor project set up with permissions so that you can create environment variables, etc.  That will have to wait until later though, as I am not in a position to do that just now.

The new project should hopefully get you going just now though.
